Gemfile

```diff
@@ -60,10 +61,18 @@ gem "mail_form"
 gem "omniauth-cas", "~> 2.0"
 gem "omniauth-rails_csrf_protection", "~> 0.1"
 gem "rsolr", "~> 2.3"
-gem "sidekiq"
+gem "sidekiq", "~> 5.0"
 gem "simple_form"
 gem "sitemap_generator", "~> 6.0"
 gem "sneakers"
 gem "twitter-typeahead-rails"
+gem "view_component", "2.57.1"
```

Comment:
Could we add a comment about why we pinned at this version since it's so specific

Reply:
Done.